src/app/services/visualize.service.ts

```diff
@@ -284,4 +286,53 @@ export class VisualizeService {
     }
     return geojsonData;
   }
+
+  public getBbox(geoJson: any): BBox2d {
+    return bbox(geoJson) as BBox2d;
+  }
+
+  public addGeocodingPreviewLayer(geoJson: any) {
+    const source = this.map.getSource(GEOCODING_PREVIEW_ID);
+    if(!source) {
+      this.map.addSource(GEOCODING_PREVIEW_ID, {
+        type: 'geojson',
+        data: geoJson
+      });
+    } else {
+      source.setData(geoJson);
+    }
+
+    const circlePaint = {
```

Comment:
I think it could be interesting to be able to easily customize those colors

Reply:
yep if we need this in the future we ll improve the implementation